skiko/src/awtMain/objectiveC/macos/MetalRedrawer.mm

```diff
@@ -115,6 +115,24 @@ static jmethodID getOnOcclusionStateChangedMethodID(JNIEnv *env, jobject redrawe
     return onOcclusionStateChanged;
 }
 
+
+static void setWindowPropertiesUnsafe(NSWindow* window, jboolean transparency) {
+    if (transparency) {
+        window.hasShadow = NO;
```

Comment:
Can it cause a blink because of scheduling? I.e. shadow is visible a moment when users start the application.

I guess it shouldn't cause a blink if `window.isVisible = true` schedules showing to the main thread as well.

Reply:
Didn't notice that. Anyway, there is no way to do it from the background thread